docker/Dockerfile

```diff
@@ -26,10 +22,26 @@ RUN mkdir /var/run/clamav && \
     chmod 750 /var/run/clamav
 
 RUN sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/clamd.conf && \
-    echo "TCPSocket 3310" >> /etc/clamav/clamd.conf
+    echo "TCPSocket 3310" >> /etc/clamav/clamd.conf && \
+    sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/freshclam.conf
 
-RUN useradd -ms /bin/bash celeryuser
-RUN usermod -a -G clamav celeryuser
+FROM parent as test
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+COPY requirements_for_test.txt requirements_for_test.txt
+RUN pip install -r requirements_for_test.txt
+
+COPY . .
+
+ADD scripts/run_celery.sh /
+
+CMD ["/run_celery.sh"]
+
+FROM parent as production
```

Comment:
is it possible to put a big line of comments or something (eg: `#######################################`) before each FROM command so it's clear where each set of commands begins?

Reply:
Yep good idea - added